release.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+go get github.com/mitchellh/gox
+
+mkdir -p release
+
+rm -rf ./release/*
+
+if [ -z "$v" ]; then
+	echo "Version number cannot be null. Run with v=[version] release.sh"
+	exit 1
+fi
+
+output="{{.Dir}}-{{.OS}}-{{.Arch}}-$v"
+osarch="!darwin/arm !darwin/arm64 !darwin/386"
+
+echo "Compiling:"
+
+os="windows linux darwin"
+arch="amd64 386 arm arm64 mips mips64 mipsle mips64le"
+
+CGO_ENABLED=0 gox -ldflags "-X main.version=${v}" -os="$os" -arch="$arch" -osarch="$osarch" -output="$output"
+mv dpiprobe-* ./release
```

Comment:
This file should be in a separate pull request, please remove it from this one. Don't mix multiple changes in one PR.